includes/in-person-payments/templates/html-in-person-payment-receipt.php

```diff
@@ -0,0 +1,191 @@
+<?php
+/**
+ * In Person Payments Print Receipt Template
+ *
+ * @package WooCommerce\Payments
+ */
+
+/**
+ * Doc format_price_helper
+ *
+ * @param  mixed $product the product.
+ * @param  mixed $currency the currency.
+ * @return mixed
+ */
+function format_price_helper( $product, $currency ) {
+	$active_price  = $product['price'];
+	$regular_price = $product['regular_price'];
+	$has_discount  = $active_price !== $regular_price;
+
+	if ( $has_discount ) {
+		return '<s>' . wc_price( $regular_price, [ 'currency' => $currency ] ) . '</s> ' . wc_price( $active_price, [ 'currency' => $currency ] );
+	}
+
+	return wc_price( $active_price, [ 'currency' => $currency ] );
+}
+
+?><!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Print Receipt</title>
+	<style>
+		body {
+			margin: 0;
+			padding: 0;
+			border: 0;
+		}
+
+		.align-left {
+			text-align: left;
+		}
+
+		.align-right {
+			text-align: right;
+		}
+		.align-top {
+			vertical-align: top;
+		}
+
+		.receipt {
+			min-width: 130px;
+			max-width: 300px;
+			padding-left: 16px;
+			padding-right: 16px;
+			margin: 0 auto;
+			text-align: center;
+			font-family: SF Pro Text, sans-serif;
```

Comment:
This font may be unavailable on Android devices.

Reply:
True @anu-rock . I added `sans-serif` as a fallback but now that you mention I think I could request an 'Android safe' equivalent font to include it here as well.